Tools/AP_Periph/Parameters.cpp

```diff
@@ -551,6 +551,15 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
   #endif
 #endif
 
+#if HAL_PERIPH_CAN_MIRROR
+    // @Param: CAN_MIRROR_PORTS
+    // @DisplayName: Which ports to mirror CAN data between
```

Comment:
slightly typo in "exist" (an extra "s").

I'd use "forwarding" rather than "mirror" because this would be consistent naming with mavlink forwarding.

I'd probably try to reword the DisplayName so it starts with CAN or forwarding.  Perhaps, "CAN forwarding between configured ports".

Reply:
Fixed the spelling, and the order of the display name.

I've kept MIRROR at the moment, because `CAN_FORWARD_PORTS` is to long, and I didn't find a shorter version I was happy with. (`FWD` is just needlessly confusing here, and I really wanted `PORTS` to be plural for the bitmask aspect). Are you strongly opinionated on this (or have a better param name I can fit in)?